syntax = "proto3";

package NSound;

import "protos/client/stream.proto"; 
import "protos/client/context.proto"; 

message TClientMessage {

    oneof Request {
        NClient.TStreamMessage stream_message = 1;
        NClient.TContextMessage context_message = 2;
    }
}

message TClientMessageStream {
    repeated TClientMessage stream = 1;
}